### PR TITLE
[#3132] Auth OS accounts only based on PAM on AIX.

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
=====

This is required for a specific sale to RoyalMail

They have an AIX system in which /etc/password is in conflict  with PAM.

This will disable /etc/passwd auth on AIX.

Change
=======

just disable for AIX

there is already a branch to add a separate PAM auth ... but we need more work to merge it

How to test
=========

reviewers: @alibotean 

check that changes make sense.

thanks

### CODE REVIEW

**Comment by adiroiban** [comment]
just a test failure due to a time delta.

needs-review

File: chevah/compat/unix_users.py [hunk 1]
```diff
@@ -217,6 +217,11 @@ def _checkPasswdFile(self, username, password):
         Return False if user was not found or password is wrong.
         Returns None if password is stored in shadow file.
         '''
+        from chevah.compat import process_capabilities
+        if process_capabilities.os_name == 'aix':
+            # AIX /etc/passwd auth is disabled for this release.
```
Comment (alibotean) [inline chevah/compat/unix_users.py]: No fixme ref ;)

**Comment by alibotean** [comment]
I suggest we add a fixme ref to know why this is was added and when we plan to remove it.

changes-approved

**Comment by adiroiban** [comment]
Thanks. I plan to revert this ASAP  and land #55 soon after this hit master

**Pull request closed without merging**